conf/messages.en

```diff
@@ -859,8 +822,6 @@ quarterChargePaid.title = In which quarter of that year was your lifetime allowa
 quarterChargePaid.heading = In which quarter of that year was your lifetime allowance charge paid?
 quarterChargePaid.hint = Select the quarter of the year your lifetime allowance charge was paid
 quarterChargePaid.aprToJul = 1 April to 30 June
-quarterChargePaid.julToOct = 1 July to 30 September
-quarterChargePaid.octToJan = 1 October to 31 December
```

Comment:
Does these definitely need removing?

Reply:
removed as not in use and we suspect it was previously half removed from our service